app/rpc/RPC.ts

```diff
@@ -235,13 +232,18 @@ export default class RPC {
   }
 
   static async rpcFetchServerHeight(): Promise<number> {
-    const info = await RPC.rpcGetInfoObject();
+    try {
+      const info = await RPC.rpcGetInfoObject();
 
-    if (info) {
-      return info.latestBlock;
-    }
+      if (info) {
+        return info.latestBlock;
+      }
 
-    return 0;
+      return 0;
+    } catch (error) {
+      console.log(`Critical Error server block height ${error}`);
+      return 0;
```

Comment:
should this return the same number in both the success and failure cases?

Reply:
both are in a way failure cases... if the object info is undefined/null return 0 and if there are any kind of Error return 0 as well... in the success case return `info.latestBlock`.